Userland/Libraries/LibWeb/Fetch/Infrastructure/HTTP/Headers.cpp

```diff
@@ -53,6 +53,22 @@ JS::NonnullGCPtr<HeaderList> HeaderList::create(JS::VM& vm)
     return vm.heap().allocate_without_realm<HeaderList>();
 }
 
+ErrorOr<Vector<ByteBuffer>> HeaderList::unique_names() const
```

Comment:
https://github.com/SerenityOS/serenity/blob/master/Documentation/Browser/Patterns.md#comments

> When adding non-standard code for a feature that is otherwise well-specified, it should be marked as such.